src/app/(frontend)/(dashboard)/(navigation)/challenges/components/recommended-courses.tsx

```
@@ -0,0 +1,156 @@
+      <CardContent>
+        <div className="flex items-center mb-4">
+          <CardTitle className="text-lg font-semibold">Recommended Courses</CardTitle>
+          <Button asChild variant="outline" size="sm" className="ml-auto">
+            <Link href="/courses">See all</Link>
+          </Button>
+        </div>
+        <div className="grid grid-cols-3 gap-4">
+          {recommendedCourses.map((course: any) => {
+            const isLocked = !course.orderedChapters || course.orderedChapters.length === 0
+            const cardContent = (
+              <Card className="relative flex flex-col items-stretch p-0 h-full">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)